client/src/views/client-area/BmsLogin.vue

```diff
@@ -191,11 +201,13 @@ import { Env } from '@/services/environment';
 const props = defineProps<{
   email?: string;
   hideHeader?: boolean;
+  hideBackButton?: boolean;
```

Comment:
As a general rule (no need to change anything here, just a note for future PRs), when adding optional parameters/props, if they are not set, the behavior of the function/component should not change. Here, if the new prop is not set, the behavior changes (breaking the API), so if you forgot to change a use of `BmsLogin` somewhere, now we've got a bug.

We don't have a lot of usage of BmsLogin so it's very probably fine here, just a general rule.